### DISCUSSION TITLE
How to configure EMQX to support both ipv4 and ipv6

### DISCUSSION THREAD

**Discussion by kmarimuthu90** [emqx/emqx#14000] [General]
I have deployed EMQX in the AKS cluster ( dual-stack which supports ipv4 and ipv6 ). 

EMQX stateful set deployment ( docker image - emqx/emqx:4.4.18
AKS node pool VM OS - Ubuntu 
AKS version 1.29.x
EMQX LB external IP is exposed in both ipv4 and ipv6. 

EMQX supports ipv4 by default but we need to support both ipv4 and ipv6 devices connection to EMQX.

**Comment by zmstone** [comment]
EMQX listeners with ipv6 enabled should be able to accept ipv4 clients. We do not set v6-only flag.

**Comment by kmarimuthu90** [comment]
Pls confirm the below,
For example, listener.ssl.external= :::8883

Will it work for both ipv4 and ipv6 traffic on same 8883 port ? And do we
need to change anything else?

Thanks
Krishnakumar
